openml/runs/functions.py

```diff
@@ -1073,4 +1075,7 @@ def format_prediction(
                 sample = 0
         probabilities = [proba[c] for c in task.class_labels]
         return [repeat, fold, sample, index, *probabilities, truth, prediction]
-    return [repeat, fold, index, truth, prediction]
+    elif isinstance(task, OpenMLRegressionTask):
+        return [repeat, fold, index, truth, prediction]
+    else:
+        raise TypeError(f"Formatting for {type(task)} is not supported.")
```

Comment:
Would a `NotImplementedError` be more logical here?

Reply:
Sure, updated.